python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -730,4 +723,4 @@ def _execution_allowed(
     ):
         return True
 
-    return global_ok_sans_memory and downstream_memory_ok
+    return False
```

Comment:
Temporarily removing because `test_runtime_metrics_scheduling` fails otherwise. Will actually disable in a separate PR